extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -148,8 +173,9 @@ function renderPassiveFeedback(video, response) {
     let filteredData = response.passive_question.filter(question => question.timestamp !== null && question.count !== null);
     let amounts = filteredData.map(question => question.count);
     let labels = filteredData.map(question => question.timestamp);
+
     console.debug(filteredData);
-    new Chart(passiveChart, {
+    let chart = new Chart(passiveChart, {
```

Comment:
let's also make `char` a global variable?

`let char = null`

I always wonder, if this variable is declared here, wouldn't the chart be destroyed by the end of the function? This just seems so odd after doing typed languages